### PR TITLE
Contribution demo prep doc

### PR DESCRIPTION
**Pull request by eyalpalo**
## Status
Ready

## Related Issues
fixes: link to the issue

## Description
A new doc with explanation about the demo.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by DeanArbel** [comment]
@jrosenpanw Please assign someone from your team to go over this document, and assist @eyalpalo to finalize it.

File: docs/contributing/demo-prep.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+## Contribution Demo
+
+Demo is the last stage of the contribution before it is merged into the internal repo.
+In order to be prepared as much as possible to the demo and avoid post-demo changes requests make sure to fo through the steps detailed int this document.
+
+### General Notes
+- The purpose of the demo is to make sure the contribution is up to XSOAR standards and to ensure everything works as expected while providing good UX. 
+- The participants who will take part of the demo are the contributor, the PR reviewer, and based on the content of the PR a security reviewer as well.
+- The demo should take up to an hour.
+
```
Comment (ShahafBenYakir) [inline docs/contributing/demo-prep.md:19]:
```suggestion

### Pre Demo
- Make sure the change requests from your code review are fully addressed and fixed.
- Prepare an XSOAR instance that has all the recent changes and has the most updated version of your pack, the demo will be performed in this environment.
```

**Review by ShahafBenYakir — COMMENTED** [review]
Great job

File: docs/contributing/demo-prep.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Demo playbooks (if applicable).
+- Review layout, incident & indicator type, incident & indicator fields and classifiers (if applicable)
+  - Verify layout is bound to incident / indicator type.
+  - Verify incident / indicator fields are bound to incident / indicator types.
+  - Verify classifier is bound to incident type.
+  - Verify playbook is bound to playbook.
+
+### Post Demo
+- If there were any changes requested during the demo by the reviewers, they must be fixed and committed.
+- After all requested changes made, the PR will be merged.
```
Comment (ShahafBenYakir) [inline docs/contributing/demo-prep.md:43]:
```suggestion
- After all requested changes are made, the PR will be merged.
```

File: docs/contributing/demo-prep.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - [Python code conventions](https://xsoar.pan.dev/docs/integrations/code-conventions)
+    - [Context and Outputs](https://xsoar.pan.dev/docs/integrations/context-and-outputs)
+    - [Context Standards](https://xsoar.pan.dev/docs/integrations/context-standards-about)
+- Demo fetch incidents (if applicable) - verification that incidents are fetched and displayed correctly.
+- Demo playbooks (if applicable).
+- Review layout, incident & indicator type, incident & indicator fields and classifiers (if applicable)
+  - Verify layout is bound to incident / indicator type.
+  - Verify incident / indicator fields are bound to incident / indicator types.
+  - Verify classifier is bound to incident type.
+  - Verify playbook is bound to playbook.
```
Comment (ShahafBenYakir) [inline docs/contributing/demo-prep.md:39]: Did you mean `... bound to an incident type`?

**Comment by ShahafBenYakir** [comment]
Need to add this page to the left toolbar and also add link in the lest section [here](https://xsoar-pan-dev--pull-request-973-p1kdgbwr.web.app/docs/contributing/contributing#after-opening-a-pull-request)

File: docs/contributing/contributing.md [hunk 4]
```diff
@@ -95,6 +95,7 @@ After opening the Pull Request, make sure that you:
 - Sign the [CLA](https://github.com/demisto/content/blob/master/docs/cla.pdf): every contributor must sign our Contributor License Agreement in order for their contribution to be added to our content. In case of CLA issues check out our [FAQs](../concepts/faq#cla-is-pending-even-though-i-signed-the-agreement).
 - Monitor your Pull Request on GitHub and be ready for a demo: our Content team will add comments to the Pull Request, asking questions and requesting changes. In order to establish a decent release process for your contribution, you are kindly asked to respond to the reviewer's code review and apply the required changes within 14 days. Stale Pull Requests might be closed.
 - At some point, we'll ask you to schedule a meeting to see an interactive demo. Make sure you have a working installation of Cortex XSOAR with your pack fully configured.
+- After all changes requested are committed, a demo will be scheduled. Make sure you understand how the demo will be conducted and get prepared. Check out our [Contribution Demo Page](../contributing/demo-prep).
```
Comment (ShahafBenYakir) [inline docs/contributing/contributing.md:98]: Look at the above point it talks exactly about demo

File: docs/contributing/contributing.md [hunk 5]
```diff
@@ -94,7 +94,7 @@ After opening the Pull Request, make sure that you:
 
 - Sign the [CLA](https://github.com/demisto/content/blob/master/docs/cla.pdf): every contributor must sign our Contributor License Agreement in order for their contribution to be added to our content. In case of CLA issues check out our [FAQs](../concepts/faq#cla-is-pending-even-though-i-signed-the-agreement).
 - Monitor your Pull Request on GitHub and be ready for a demo: our Content team will add comments to the Pull Request, asking questions and requesting changes. In order to establish a decent release process for your contribution, you are kindly asked to respond to the reviewer's code review and apply the required changes within 14 days. Stale Pull Requests might be closed.
-- At some point, we'll ask you to schedule a meeting to see an interactive demo. Make sure you have a working installation of Cortex XSOAR with your pack fully configured.
+- At some point, we'll ask you to schedule a meeting to see an interactive demo. Make sure you have a working installation of Cortex XSOAR with your pack fully configured. Check out our [Contribution Demo Page](../contributing/demo-prep).
```
Comment (ShahafBenYakir) [inline docs/contributing/contributing.md:97]:
```suggestion
- At some point, we'll ask you to schedule a meeting to see an interactive demo. Make sure you have a working installation of Cortex XSOAR with your pack fully configured. Check out our [Contribution Demo Page](../contributing/demo-prep) for more details.
```

File: docs/contributing/demo-prep.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+---
+id: demo_prep
```
Comment (ShahafBenYakir) [inline docs/contributing/demo-prep.md:2]: There is a mismatch between this and the id in the sidebar

**Review by ShahafBenYakir — APPROVED** [review]
Nice

**Comment by ShirleyDenkberg** [comment]
@eyalpalo I could not edit the README directly in this PR, so I sent you the file that I reviewed and edited by Slack.

**Pull request merged**